docs/content/en/docs/Advanced/coco.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Confidential computing setup"
+linkTitle: "Confidential Computing"
+weight: 9
+description: >
+---
+
```

Comment:
we might want to add an alert here, as this is currently experimental and calls for optimizations in many places still

Reply:
true, will add an alert